samples/matter/lock/README.rst

```diff
@@ -73,6 +73,27 @@ For details, see the `Commissioning the device`_ section.
 
 .. matter_door_lock_sample_remote_testing_end
 
+Door lock credentials
+=====================
+
+The application supports PIN code credentials.
```

Comment:
Are we able to support other types of credentials or only PIN codes? I mean, I know that application uses pin codes, but the credentials module is able to support other types as well, right? If so, maybe we should write it somewhere that we present only pin codes in our app, but it could be also changed to use something different.

Reply:
Sure, good point.